templates/quarkus/events/README.md

```diff
@@ -2,7 +2,7 @@
 
 Welcome to your new Quarkus function project!
 
-This sample project contains single function: `functions.Function.echo()`,
+This sample project contains single function: `functions.Function.function()`,
```

Comment:
`functions.Function.function()` this looks odd, but I don't know what would be better.

Reply:
Possibly `functions.Function.fn()` or `functions.Function.invoke()`?